src/js/routes/development.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
```

Comment:
is it hard to make this file TS?

Reply:
We would need to add types to the entire foundation ui package, so I cut the scope here

Reply:
i must be missing something here. i meant to ask about converting `src/js/routes/development.js` to TS. i don't understand how `foundation-ui` is an issue yet. (also we can choose to `// @ts-ignore` some imports for now, if that's what your concerns are about - it's still better than not typing the file at all IMHO)